feat: Creating an example of an email summary (#15)
llamaapi/llama-api-docs: essentials/function.mdx

```
It´s currently moderate rain in London with a temperature of 8.0 degrees Celsius.
```

> ### Example 4: `get_email_resume`

Reviewer: @fuchsfelipel , what do you think of the idea of ​​function calling having just one example that is already very complete and adding the other examples to a page that only contains examples ? So each example can be better detailed on this examples page

Reviewer: Could be on another task

Reviewer 2: I agree! In fact, for this specific example, I'd move it to a page called `Tutorial` and give a step-by-step explanation

    try:
        mail = imaplib.IMAP4_SSL("imap.gmail.com")
        mail.login(login, password)

        mail.select("inbox")

        status, messages = mail.search(None, "(UNSEEN)")

        email_ids = messages[0].split()[::-1]

        limite = min(quantity, len(email_ids))

        for i in range(limite):
            email_id = email_ids[i]
            _, msg_data = mail.fetch(email_id, "(RFC822)")
            for response_part in msg_data:
                if isinstance(response_part, tuple):
                    email_message = email.message_from_bytes(response_part[1])
                    subject, encoding = decode_header(email_message["Subject"])[0]
                    if isinstance(subject, bytes):
                        subject = subject.decode(encoding or "utf-8")

                    sender = email_message.get("From")

            email_information.append({"Subject": subject, "From": sender})
                    

        mail.logout()
        return json.dumps(email_information)
    except Exception as e:
        print("Err:", str(e))
```
So, we send the emails to a new function that will summarize

```python
summarizedResponse = {

>   "model": "codellama-7b-instruct",

Reviewer: Why are you using two different models?

Author: Corrected to use only one model

```
It´s currently moderate rain in London with a temperature of 8.0 degrees Celsius.
```
### Example 4: `get_email_resume`
For this example we will use gmail as an email service
- Objective: Create a resume from your e-mails
- Parameter: `value` (desired quantity of e-mails), `login` (your e-mail), `password` (your e-mail password)
- Usage Example: User provides a location and receive local weather information

> 

Reviewer 2: When you move this to the `Tutorial` page, give an explanation of how it works:

Eg. On the code below, what actually happens is:
1. We send the request to the LLM
2. The LLM sees your emails using IMAP
3. ....

### Request
Firstly, you need to create a password for less secure apps by following the link: https://support.google.com/a/answer/6260879?hl=en.
With the password created, we can make the first request to extract the email, password and number of emails that will be summarized.
```python
api_request_json = {
  "model": "llama-70b-chat",
  "messages": [

>     {"role": "user", "content": "Make a resume from my last e-mail, login: tiagovictordeoliveira@gmail.com and password: nnmy fvbx sucz gxdu"},

Reviewer 2: This is a big security problem.... your email and password are now public to the world! I imagine you already expired your password... but change your email so you don't get too much SPAM messages

Reviewer 2: https://stackoverflow.com/questions/2110595/redaction-in-git

>     {"role": "user", "content": "Make a resume from my last e-mail, login: tiagovictordeoliveira@gmail.com and password: nnmy fvbx sucz gxdu"},

Reviewer 2: ```suggestion
    {"role": "user", "content": "Make a summary of my last e-mail, login: tiagovictordeoliveira@gmail.com and password: nnmy fvbx sucz gxdu"},
```

  ],
  "functions": [
    {

>         "name": "get_email_resume",

Reviewer 2: Rename to `get_email_summary`

In english:
`resume` quer dizer `continuar``
`summary` quer dizer `resumo`
